plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -8,21 +8,16 @@ import { useState, useEffect, useRef } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { MIN_TITLE_LENGTH } from '../constants';
+import { MIN_TITLE_LENGTH_FOR_DESCRIPTION } from '../constants';
 import { WriteItForMeBtn, StopCompletionBtn } from '../components';
 import { useTinyEditor, useCompletion, useFeedbackSnackbar } from '../hooks';
-import { recordTracksFactory } from '../utils';
+import { recordTracksFactory, getPostId } from '../utils';
 
 const DESCRIPTION_MAX_LENGTH = 300;
 
-const getPostId = () =>
-	Number(
-		( document.querySelector( '#post_ID' ) as HTMLInputElement )?.value
-	);
-
 const getApiError = () => {
 	return __(
-		`Apologies, this is an experimental feature and there was an error with this service. Please try again.`,
+		`❗ We're currently experiencing high demand for our experimental feature. Please check back in shortly!`,
```

Comment:
We already have the exclamation point at the beginning so lets make change the last one to a period `Please check back in shortly.`